keps/sig-scheduling/1258-default-pod-topology-spread/README.md

```diff
@@ -359,14 +367,24 @@ To ensure this feature to be rolled out in high quality. Following tests are man
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
   fields of API types, flags, etc.?**
 
-  TBD for GA.
+  No.
 
 ### Monitoring Requirements
 
 * **How can an operator determine if the feature is in use by workloads?**
 
   All Pods are affected, unless they have explicit spreading constraints (.spec.topologySpreadConstraints).
 
+* **How can someone using this feature know that it is working for their instance?**
+
+  - [ ] Events
+    - Event Reason: 
+  - [X] API .status
+    - Condition name: pods that don't provide any `pod.spec.topologySpreadConstraints` still have default `pod.spec.topologySpreadConstraints`.
```

Comment:
I don't think this is a good answer to the question. However, we could improve the PodScheduled event message to include the top scores. A similar suggestion was raised here https://github.com/kubernetes/enhancements/pull/3116/files#r780402728

Reply:
@ahg-g what do you think if we this improvement as graduation criteria for this feature?

Reply:
I wouldn't block on it. Lets track it separately.

Reply:
https://github.com/kubernetes/kubernetes/issues/107556

Reply:
@alculquicondor what should we put here for now?

Reply:
Other: observe the scheduled pods and verify the spreading is satisfied.

So, basically, there is no easy way.

Reply:
updated.